server/lib/config/nginx.conf.example

```diff
@@ -0,0 +1,27 @@
+# Nginx setup need to done separately on staging and production.
```

Comment:
Make it into a template and add an ansible role to do the setup?

Reply:
Oh now that I think about it I think I need to update this. @ndokos I dont think we need separate Nginx setup on staging or production (since we only have one Nginx instance running and we front staging and production pbench server by this one instance, correct me if I am wrong @portante ). So all we need to do is update that nginx config so it can route the calls to appropriate servers (if its not done yet). So we dont need any Nginx setup on server itself per se

Reply:
I think the real question here, for distribution, is whether we're going to require anyone standing up a Pbench server to supply and configure their own NGINX instance (in which case we should at least produce a sample configuration file) or whether we offer (perhaps optionally) to stand up a dedicated local NGINX instance for installations that wouldn't otherwise have one....

Reply:
@dbutenhof yes, and this example config will let someone do exactly that, that is configure their own NGINX instance with their pbench server

Reply:
So that first comment line should go, right?